src/index.js

```diff
@@ -14,7 +14,7 @@
  */
 import naclFactory from 'js-nacl';
 import api from './api/liskApi';
-import crypto from './crypto';
+import crypto from './cryptography';
```

Comment:
I think it would be great, also to omit future confusion with node's native `crypto` module.

Reply:
other projects that are using this would be affected, but is that ok?

Reply:
We are about to release 1.0.0 - therefore all the changes that make a lot of sense and are breaking are better fit to be done now rather than later actually.